schemas/bashly.json

```
@@ -74,7 +74,8 @@
                     ]
                 }
             },
-            "additionalProperties": false
+            "additionalProperties": false,
+            "patternProperties": { "^x-": {} }
```

```suggestion
            "patternProperties": { "^x-.": {} }
```

It's better to use `.` not to allow properties consisting of just prefix `x-`.

---

I don't understand.

---

Sure. I have updated the regex, feel free to push to this branch with your updates.

Example would be providing another "help" for a command, flag or arg, that will be used for man page generation:

```yaml
commands:
- name: status
  help: Show status
  x-manpage-help: |-
    This command shows the status of all `images`, `containers` and `volumes`.
    Use with `--verbose` to see deleted containers.
```